### PR TITLE
Add a knowledge learn from the team into the doc for testing

### PR DESCRIPTION
**Pull request by schenedx**
@clintonb See this update to the testing doc.

### CODE REVIEW

File: docs/testing.rst [hunk 1]
```diff
@@ -7,6 +7,18 @@ To run the unit test suite followed by quality checks, run:
 
     $ make validate
 
+If you run into the following error:
+
+.. code-block:: bash
+
+    OfflineGenerationError: You have offline compression enabled but key "619946fd97399c648d78132993aeab02" is missing from offline manifest. You may need to run "python manage.py compress".
```
Comment (clintonb) [inline docs/testing.rst]: Rephrase this to tell folks how to avoid the error. Place a paragraph above the `make validate` command.

```
The tests use the `ecommerce.settings.test` settings module, which has compression enabled. Before running the tests, run the command below to compile and compress static files.
```

Comment (schenedx) [inline docs/testing.rst]: @clintonb Thanks for the paragraph. I have made the change. Please review again.

**Comment by rlucioni** [comment]
@srpearce FYI.

**Comment by clintonb** [comment]
:+1:
